docs/user/README.md

```diff
@@ -118,6 +118,31 @@ local instance of `modelardbd` through a REPL, simply run `modelardb`:
 modelardb
 ```
 
+Be aware that the REPL currently does not support splitting SQL statements over multiple lines and that the SQL
+statements do not need to end with a `;`. In addition to SQL statements, the REPL also supports listing all tables
+using `\dt` and printing the schema of a table using`\d table_name`:
```

Comment:
Very minor but it seems that a space is missing after `using`.

Reply:
Fixed.